ambry-api/src/main/java/com.github.ambry/clustermap/ClusterParticipant.java

```diff
@@ -67,6 +67,12 @@
   void registerPartitionStateChangeListener(StateModelListenerType listenerType,
       PartitionStateChangeListener partitionStateChangeListener);
 
+  /**
+   * @return {@link ReplicaSyncUpService} that is used to determine new replica has caught up with peers or peer replicas
```

Comment:
minor: lets also add a line for the method comment saying "Gets the {@link ReplicaSyncUpService} object"

Reply:
added